syntax = "proto3";

package execution;

service Execute {
    rpc execute_command( stream ExecuteRequest) returns (stream ExecutionResponse) {}
}

message ExecuteRequest {
    string container_name = 1;
    string command = 2;
    string dir = 3;
    string stdin = 4;
}

message ExecutionResponse {
    string message = 1;
}
